zerver/tests/webhooks/test_greenhouse.py

```diff
@@ -0,0 +1,69 @@
+                            "**Attachments:**\n[Resume](https://prod-heroku.s3.amazonaws.com/...)\n")
+
+        self.send_and_test_stream_message('candidate_hired',
+                                          expected_subject,
+                                          expected_message,
+                                          content_type="application/x-www-form-urlencoded")
+
+    def test_message_candidate_rejected(self):
+        # type: () -> None
+        expected_subject = "Reject Candidate - 265788"
+        expected_message = ("Reject Candidate\n>Hector Porter\nID: "
+                            "265788\nApplying for role:\nDesigner"
+                            "\n**Emails:**\nPersonal\n"
+                            "hector.porter.265788@example.com\n\n\n>"
+                            "**Attachments:**\n[Resume](https://prod-heroku.s3.amazonaws.com/...)\n")
+
+        self.send_and_test_stream_message('candidate_rejected',
+                                          expected_subject,
+                                          expected_message,
+                                          content_type="application/x-www-form-urlencoded")
+
+    def test_message_candidate_stage_change(self):
+        # type: () -> None
+        expected_subject = "Candidate Stage Change - 265772"
+        expected_message = ("Candidate Stage Change\n>Giuseppe Hurley"
+                            "\nID: 265772\nApplying for role:\n"
+                            "Designer\n**Emails:**\nPersonal"
+                            "\ngiuseppe.hurley@example.com\n\n\n>"
+                            "**Attachments:**\n[Resume](https://prod-heroku.s3.amazonaws.com/...)"
+                            "\n[Cover_Letter](https://prod-heroku.s3.amazonaws.com/...)"
+                            "\n[Attachment](https://prod-heroku.s3.amazonaws.com/...)\n")
+
+        self.send_and_test_stream_message('candidate_stage_change',
+                                          expected_subject,
+                                          expected_message,
+                                          content_type="application/x-www-form-urlencoded")
+
+    def test_message_prospect_created(self):
+        # type: () -> None
+        expected_subject = "New Prospect Application - 968190"
+        expected_message = ("New Prospect Application\n>Trisha Troy"
+                            "\nID: 968190\nApplying for role:\n"
+                            "Designer\n**Emails:**\nPersonal"
+                            "\nt.troy@example.com\n\n\n>**Attachments:**"
+                            "\n[Resume](https://prod-heroku.s3.amazonaws.com/...)\n")
+
+        self.send_and_test_stream_message('prospect_created',
+                                          expected_subject,
+                                          expected_message,
+                                          content_type="application/x-www-form-urlencoded")
```

Comment:
Could we have `CONTENT_TYPE` class attr - it'd be easier if we'd like to change it in the future.